### PR TITLE
handlers: fixed duplicated variable

### PR DESCRIPTION
**Pull request by giautm**
Close: https://github.com/ogen-go/ogen/issues/662

### CODE REVIEW

File: internal/integration/sample_api/oas_handlers_gen.go [hunk 1]
```diff
@@ -2164,8 +2164,7 @@ func (s *Server) handleSecurityTestRequest(args [0]string, w http.ResponseWriter
 			ID:   "securityTest",
 		}
 	)
-	sctx, err := s.securityAPIKey(ctx, "SecurityTest", r)
-	if err != nil {
```
Comment (tdakkota) [inline internal/integration/sample_api/oas_handlers_gen.go:2168]: Usually, we change all generated files in a one separate commit. Could you move these changes to a separate commit, please?

**Review by tdakkota — CHANGES_REQUESTED** [review]

File: internal/integration/sample_api/oas_handlers_gen.go (hunk 1 above)
Comment (giautm) [reply internal/integration/sample_api/oas_handlers_gen.go:2168]: Done @tdakkota

**Review by tdakkota — APPROVED** [review]
LGTM. Thank you for sending PR!
